openedx_external_enrollments/admin.py

```diff
@@ -82,3 +83,19 @@ class ExternalEnrollmentAdmin(ReadOnlyAdminMixin, admin.ModelAdmin):
     ]
 
     search_fields = ('controller_name', 'course_shell__id', 'email')
+
+
+@admin.register(PathstreamEnrollment)
+class PathstreamEnrollmentAdmin(admin.ModelAdmin):
```

Comment:
@Jacatove, Do you think it is necessary to add this to the admin site?

Reply:
@ivanvgh now that we are not using this model, I will delete this file and the model.